### PR TITLE
[chore] bump util to v0.2

### PR DESCRIPTION
**Pull request by joshuajeschek**

### CODE REVIEW

File: src/test/java/github/weichware10/analyse/VerlaufTest.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-import github.weichware10.util.Data;
+import github.weichware10.util.data.TrialData;
 import github.weichware10.util.Enums.ToolType;
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/VerlaufTest.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Enums.ToolType' import. Should be before 'github.weichware10.util.data.TrialData'.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-import github.weichware10.util.Data;
+import github.weichware10.util.data.TrialData;
 import github.weichware10.util.Enums.ToolType;
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Enums.ToolType' import. Should be before 'github.weichware10.util.data.TrialData'.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 3]
```diff
@@ -15,15 +15,15 @@
      */
     @Test
     public void createHeatmapShouldWork() {
-        Heatmap hm1 = new Heatmap(new Data(ToolType.ZOOMMAPS, 1), new ConfigHeatmap());
+        Heatmap hm1 = new Heatmap(new TrialData(ToolType.ZOOMMAPS, "1", "3"), new ConfigHeatmap());
         assertEquals("createHeatMap sollte .../heatmap/HEATMAP_ZOOMMAPS_1.jpg ausgeben",
                 ".../heatmap/HEATMAP_ZOOMMAPS_1.jpg", hm1.createHeatmap());
 
-        Heatmap hm2 = new Heatmap(new Data(ToolType.EYETRACKING, 7), new ConfigHeatmap());
+        Heatmap hm2 = new Heatmap(new TrialData(ToolType.EYETRACKING, "7", "3"), new ConfigHeatmap());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 102).

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 4]
```diff
@@ -15,15 +15,15 @@
...
         assertEquals("createHeatMap sollte .../heatmap/HEATMAP_EYETRACKING_7.jpg ausgeben",
                 ".../heatmap/HEATMAP_EYETRACKING_7.jpg", hm2.createHeatmap());
 
-        Heatmap hm3 = new Heatmap(new Data(ToolType.CODECHARTS, 3), new ConfigHeatmap());
+        Heatmap hm3 = new Heatmap(new TrialData(ToolType.CODECHARTS, "3", "3"), new ConfigHeatmap());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 101).

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 5]
```diff
@@ -33,22 +33,22 @@ public void createHeatmapShouldWork() {
      */
     @Test
     public void compHeatmapsShouldWork() {
-        Heatmap hm1 = new Heatmap(new Data(ToolType.CODECHARTS, 4), new ConfigHeatmap());
-        Heatmap hm2 = new Heatmap(new Data(ToolType.EYETRACKING, 2), new ConfigHeatmap());
+        Heatmap hm1 = new Heatmap(new TrialData(ToolType.CODECHARTS, "4", "3"), new ConfigHeatmap());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 101).

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 6]
```diff
@@ -33,22 +33,22 @@ public void createHeatmapShouldWork() {
...
+        Heatmap hm2 = new Heatmap(new TrialData(ToolType.EYETRACKING, "2", "3"), new ConfigHeatmap());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 102).

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 7]
```diff
@@ -33,22 +33,22 @@ public void createHeatmapShouldWork() {
...
         assertEquals(
                 "compHeatmaps sollte .../heatmap/COMPHEATMAP_CODECHARTS_1_EYETRACKING_2.jpg ausgeb",
                 ".../heatmap/COMPHEATMAP_CODECHARTS_4_EYETRACKING_2.jpg",
                 Heatmap.compHeatmaps(hm1, hm2));
 
-        Heatmap hm3 = new Heatmap(new Data(ToolType.ZOOMMAPS, 1), new ConfigHeatmap());
-        Heatmap hm4 = new Heatmap(new Data(ToolType.CODECHARTS, 6), new ConfigHeatmap());
+        Heatmap hm3 = new Heatmap(new TrialData(ToolType.ZOOMMAPS, "1", "3"), new ConfigHeatmap());
+        Heatmap hm4 = new Heatmap(new TrialData(ToolType.CODECHARTS, "6", "3"), new ConfigHeatmap());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 101).

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 8]
```diff
@@ -33,22 +33,22 @@ public void createHeatmapShouldWork() {
...
         assertEquals(
                 "compHeatmaps sollte .../heatmap/COMPHEATMAP_ZOOMMAPS_1_CODECHARTS_6.jpg ausgeben",
                 ".../heatmap/COMPHEATMAP_ZOOMMAPS_1_CODECHARTS_6.jpg",
                 Heatmap.compHeatmaps(hm3, hm4));
 
-        Heatmap hm5 = new Heatmap(new Data(ToolType.EYETRACKING, 8), new ConfigHeatmap());
-        Heatmap hm6 = new Heatmap(new Data(ToolType.ZOOMMAPS, 12), new ConfigHeatmap());
+        Heatmap hm5 = new Heatmap(new TrialData(ToolType.EYETRACKING, "8", "3"), new ConfigHeatmap());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 102).

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 9]
```diff
@@ -2,7 +2,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-import github.weichware10.util.Data;
+import github.weichware10.util.data.TrialData;
 import github.weichware10.util.Enums.ToolType;
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java:5]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Enums.ToolType' import. Should be before 'github.weichware10.util.data.TrialData'.

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 10]
```diff
@@ -15,19 +15,19 @@
      */
     @Test
     public void createRelFreqImgAreaShouldWork() {
-        Diagramm dia1 = new Diagramm(new Data(ToolType.ZOOMMAPS, 1), new ConfigDiagramm());
+        Diagramm dia1 = new Diagramm(new TrialData(ToolType.ZOOMMAPS, "1", "3"), new ConfigDiagramm());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 103).

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 11]
```diff
@@ -15,19 +15,19 @@
...
         assertEquals(
                 "createRelFreqImgAre sollte .../diagramm/RELFRQIMGAREA_ZOOMMAPS_1 zurückgeben",
                 ".../diagramm/RELFRQIMGAREA_ZOOMMAPS_1.jpg",
                 dia1.createRelFreqImgArea());
 
-        Diagramm dia2 = new Diagramm(new Data(ToolType.EYETRACKING, 5), new ConfigDiagramm());
+        Diagramm dia2 = new Diagramm(new TrialData(ToolType.EYETRACKING, "5", "3"), new ConfigDiagramm());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 106).

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 12]
```diff
@@ -15,19 +15,19 @@
...
         assertEquals(
                 "createRelFreqImgAre sollte .../diagramm/RELFRQIMGAREA_EYETRACKING_5 zurückgeben",
                 ".../diagramm/RELFRQIMGAREA_EYETRACKING_5.jpg",
                 dia2.createRelFreqImgArea());
 
-        Diagramm dia3 = new Diagramm(new Data(ToolType.CODECHARTS, 132), new ConfigDiagramm());
+        Diagramm dia3 = new Diagramm(new TrialData(ToolType.CODECHARTS, "132", "3"), new ConfigDiagramm());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 107).

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 13]
```diff
@@ -39,18 +39,18 @@ public void createRelFreqImgAreaShouldWork() {
      */
     @Test
     public void createViewTimeDistrSouldWork() {
-        Diagramm dia1 = new Diagramm(new Data(ToolType.ZOOMMAPS, 1), new ConfigDiagramm());
+        Diagramm dia1 = new Diagramm(new TrialData(ToolType.ZOOMMAPS, "1", "3"), new ConfigDiagramm());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 103).

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 14]
```diff
@@ -39,18 +39,18 @@ public void createRelFreqImgAreaShouldWork() {
...
         assertEquals("createViewTimeDistr sollte .../diagramm/VIEWTIMEDISTR_ZOOMMAPS_1 zurückgeben",
                 ".../diagramm/VIEWTIMEDISTR_ZOOMMAPS_1.jpg",
                 dia1.createViewTimeDistr());
 
-        Diagramm dia2 = new Diagramm(new Data(ToolType.EYETRACKING, 5), new ConfigDiagramm());
+        Diagramm dia2 = new Diagramm(new TrialData(ToolType.EYETRACKING, "5", "3"), new ConfigDiagramm());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 106).

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 15]
```diff
@@ -39,18 +39,18 @@ public void createRelFreqImgAreaShouldWork() {
...
         assertEquals(
                 "createViewTimeDistr sollte .../diagramm/VIEWTIMEDISTR_EYETRACKING_5 zurückgeben",
                 ".../diagramm/VIEWTIMEDISTR_EYETRACKING_5.jpg",
                 dia2.createViewTimeDistr());
 
-        Diagramm dia3 = new Diagramm(new Data(ToolType.CODECHARTS, 132), new ConfigDiagramm());
+        Diagramm dia3 = new Diagramm(new TrialData(ToolType.CODECHARTS, "132", "3"), new ConfigDiagramm());
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 107).

File: src/main/java/github/weichware10/analyse/Client.java [hunk 16]
```diff
@@ -1,6 +1,6 @@
 package github.weichware10.analyse;
 
-import github.weichware10.util.Data;
+import github.weichware10.util.data.TrialData;
 import github.weichware10.util.Enums.ToolType;
```
Comment (weichbot10) [inline src/main/java/github/weichware10/analyse/Client.java:3]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Enums.ToolType' import. Should be before 'github.weichware10.util.data.TrialData'.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 17]
```diff
@@ -15,15 +15,21 @@
      */
     @Test
     public void createHeatmapShouldWork() {
-        Heatmap hm1 = new Heatmap(new Data(ToolType.ZOOMMAPS, 1), new ConfigHeatmap());
+        Heatmap hm1 = new Heatmap(
+                new TrialData(ToolType.ZOOMMAPS, "1", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 18]
```diff
@@ -15,15 +15,21 @@
...
-        Heatmap hm1 = new Heatmap(new Data(ToolType.ZOOMMAPS, 1), new ConfigHeatmap());
+        Heatmap hm1 = new Heatmap(
+                new TrialData(ToolType.ZOOMMAPS, "1", "3"), 
+                new ConfigHeatmap());
         assertEquals("createHeatMap sollte .../heatmap/HEATMAP_ZOOMMAPS_1.jpg ausgeben",
                 ".../heatmap/HEATMAP_ZOOMMAPS_1.jpg", hm1.createHeatmap());
 
-        Heatmap hm2 = new Heatmap(new Data(ToolType.EYETRACKING, 7), new ConfigHeatmap());
+        Heatmap hm2 = new Heatmap(
+                new TrialData(ToolType.EYETRACKING, "7", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 19]
```diff
@@ -15,15 +15,21 @@
...
-        Heatmap hm2 = new Heatmap(new Data(ToolType.EYETRACKING, 7), new ConfigHeatmap());
+        Heatmap hm2 = new Heatmap(
+                new TrialData(ToolType.EYETRACKING, "7", "3"), 
+                new ConfigHeatmap());
         assertEquals("createHeatMap sollte .../heatmap/HEATMAP_EYETRACKING_7.jpg ausgeben",
                 ".../heatmap/HEATMAP_EYETRACKING_7.jpg", hm2.createHeatmap());
 
-        Heatmap hm3 = new Heatmap(new Data(ToolType.CODECHARTS, 3), new ConfigHeatmap());
+        Heatmap hm3 = new Heatmap(
+                new TrialData(ToolType.CODECHARTS, "3", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 20]
```diff
@@ -33,22 +39,34 @@ public void createHeatmapShouldWork() {
      */
     @Test
     public void compHeatmapsShouldWork() {
-        Heatmap hm1 = new Heatmap(new Data(ToolType.CODECHARTS, 4), new ConfigHeatmap());
-        Heatmap hm2 = new Heatmap(new Data(ToolType.EYETRACKING, 2), new ConfigHeatmap());
+        Heatmap hm1 = new Heatmap(
+                new TrialData(ToolType.CODECHARTS, "4", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 21]
```diff
@@ -33,22 +39,34 @@ public void createHeatmapShouldWork() {
      */
     @Test
     public void compHeatmapsShouldWork() {
-        Heatmap hm1 = new Heatmap(new Data(ToolType.CODECHARTS, 4), new ConfigHeatmap());
-        Heatmap hm2 = new Heatmap(new Data(ToolType.EYETRACKING, 2), new ConfigHeatmap());
+        Heatmap hm1 = new Heatmap(
+                new TrialData(ToolType.CODECHARTS, "4", "3"), 
+                new ConfigHeatmap());
+        Heatmap hm2 = new Heatmap(
+                new TrialData(ToolType.EYETRACKING, "2", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 22]
```diff
@@ -33,22 +39,34 @@ public void createHeatmapShouldWork() {
...
+                new ConfigHeatmap());
         assertEquals(
                 "compHeatmaps sollte .../heatmap/COMPHEATMAP_CODECHARTS_1_EYETRACKING_2.jpg ausgeb",
                 ".../heatmap/COMPHEATMAP_CODECHARTS_4_EYETRACKING_2.jpg",
                 Heatmap.compHeatmaps(hm1, hm2));
 
-        Heatmap hm3 = new Heatmap(new Data(ToolType.ZOOMMAPS, 1), new ConfigHeatmap());
-        Heatmap hm4 = new Heatmap(new Data(ToolType.CODECHARTS, 6), new ConfigHeatmap());
+        Heatmap hm3 = new Heatmap(
+                new TrialData(ToolType.ZOOMMAPS, "1", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 23]
```diff
@@ -33,22 +39,34 @@ public void createHeatmapShouldWork() {
...
                 ".../heatmap/COMPHEATMAP_CODECHARTS_4_EYETRACKING_2.jpg",
                 Heatmap.compHeatmaps(hm1, hm2));
 
-        Heatmap hm3 = new Heatmap(new Data(ToolType.ZOOMMAPS, 1), new ConfigHeatmap());
-        Heatmap hm4 = new Heatmap(new Data(ToolType.CODECHARTS, 6), new ConfigHeatmap());
+        Heatmap hm3 = new Heatmap(
+                new TrialData(ToolType.ZOOMMAPS, "1", "3"), 
+                new ConfigHeatmap());
+        Heatmap hm4 = new Heatmap(
+                new TrialData(ToolType.CODECHARTS, "6", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 24]
```diff
@@ -33,22 +39,34 @@ public void createHeatmapShouldWork() {
...
+                new ConfigHeatmap());
         assertEquals(
                 "compHeatmaps sollte .../heatmap/COMPHEATMAP_ZOOMMAPS_1_CODECHARTS_6.jpg ausgeben",
                 ".../heatmap/COMPHEATMAP_ZOOMMAPS_1_CODECHARTS_6.jpg",
                 Heatmap.compHeatmaps(hm3, hm4));
 
-        Heatmap hm5 = new Heatmap(new Data(ToolType.EYETRACKING, 8), new ConfigHeatmap());
-        Heatmap hm6 = new Heatmap(new Data(ToolType.ZOOMMAPS, 12), new ConfigHeatmap());
+        Heatmap hm5 = new Heatmap(
+                new TrialData(ToolType.EYETRACKING, "8", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/HeatmapTest.java [hunk 25]
```diff
@@ -33,22 +39,34 @@ public void createHeatmapShouldWork() {
...
                 ".../heatmap/COMPHEATMAP_ZOOMMAPS_1_CODECHARTS_6.jpg",
                 Heatmap.compHeatmaps(hm3, hm4));
 
-        Heatmap hm5 = new Heatmap(new Data(ToolType.EYETRACKING, 8), new ConfigHeatmap());
-        Heatmap hm6 = new Heatmap(new Data(ToolType.ZOOMMAPS, 12), new ConfigHeatmap());
+        Heatmap hm5 = new Heatmap(
+                new TrialData(ToolType.EYETRACKING, "8", "3"), 
+                new ConfigHeatmap());
+        Heatmap hm6 = new Heatmap(
+                new TrialData(ToolType.ZOOMMAPS, "12", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/HeatmapTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 26]
```diff
@@ -15,19 +15,25 @@
      */
     @Test
     public void createRelFreqImgAreaShouldWork() {
-        Diagramm dia1 = new Diagramm(new Data(ToolType.ZOOMMAPS, 1), new ConfigDiagramm());
+        Diagramm dia1 = new Diagramm(
+                new TrialData(ToolType.ZOOMMAPS, "1", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 27]
```diff
@@ -15,19 +15,25 @@
...
+                new TrialData(ToolType.ZOOMMAPS, "1", "3"), 
+                new ConfigDiagramm());
         assertEquals(
                 "createRelFreqImgAre sollte .../diagramm/RELFRQIMGAREA_ZOOMMAPS_1 zurückgeben",
                 ".../diagramm/RELFRQIMGAREA_ZOOMMAPS_1.jpg",
                 dia1.createRelFreqImgArea());
 
-        Diagramm dia2 = new Diagramm(new Data(ToolType.EYETRACKING, 5), new ConfigDiagramm());
+        Diagramm dia2 = new Diagramm(
+                new TrialData(ToolType.EYETRACKING, "5", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 28]
```diff
@@ -15,19 +15,25 @@
...
+                new TrialData(ToolType.EYETRACKING, "5", "3"), 
+                new ConfigDiagramm());
         assertEquals(
                 "createRelFreqImgAre sollte .../diagramm/RELFRQIMGAREA_EYETRACKING_5 zurückgeben",
                 ".../diagramm/RELFRQIMGAREA_EYETRACKING_5.jpg",
                 dia2.createRelFreqImgArea());
 
-        Diagramm dia3 = new Diagramm(new Data(ToolType.CODECHARTS, 132), new ConfigDiagramm());
+        Diagramm dia3 = new Diagramm(
+                new TrialData(ToolType.CODECHARTS, "132", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 29]
```diff
@@ -39,18 +45,24 @@ public void createRelFreqImgAreaShouldWork() {
      */
     @Test
     public void createViewTimeDistrSouldWork() {
-        Diagramm dia1 = new Diagramm(new Data(ToolType.ZOOMMAPS, 1), new ConfigDiagramm());
+        Diagramm dia1 = new Diagramm(
+                new TrialData(ToolType.ZOOMMAPS, "1", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 30]
```diff
@@ -39,18 +45,24 @@ public void createRelFreqImgAreaShouldWork() {
...
+        Diagramm dia1 = new Diagramm(
+                new TrialData(ToolType.ZOOMMAPS, "1", "3"), 
+                new ConfigDiagramm());
         assertEquals("createViewTimeDistr sollte .../diagramm/VIEWTIMEDISTR_ZOOMMAPS_1 zurückgeben",
                 ".../diagramm/VIEWTIMEDISTR_ZOOMMAPS_1.jpg",
                 dia1.createViewTimeDistr());
 
-        Diagramm dia2 = new Diagramm(new Data(ToolType.EYETRACKING, 5), new ConfigDiagramm());
+        Diagramm dia2 = new Diagramm(
+                new TrialData(ToolType.EYETRACKING, "5", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

File: src/test/java/github/weichware10/analyse/DiagrammTest.java [hunk 31]
```diff
@@ -39,18 +45,24 @@ public void createRelFreqImgAreaShouldWork() {
...
+                new TrialData(ToolType.EYETRACKING, "5", "3"), 
+                new ConfigDiagramm());
         assertEquals(
                 "createViewTimeDistr sollte .../diagramm/VIEWTIMEDISTR_EYETRACKING_5 zurückgeben",
                 ".../diagramm/VIEWTIMEDISTR_EYETRACKING_5.jpg",
                 dia2.createViewTimeDistr());
 
-        Diagramm dia3 = new Diagramm(new Data(ToolType.CODECHARTS, 132), new ConfigDiagramm());
+        Diagramm dia3 = new Diagramm(
+                new TrialData(ToolType.CODECHARTS, "132", "3"),
```
Comment (weichbot10) [inline src/test/java/github/weichware10/analyse/DiagrammTest.java]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line has trailing whitespaces.

**Comment by Rotwolf** [comment]
Dear weichbot10,
with the highest emotion and explicit statement,
I would like to share with you,
That I hate you.

Yours sincerely, im in tears,
Jonathan

**Comment by joshuajeschek** [comment]
@Rotwolf macht einen neuen PR, da es ja seine Arbeit ist

**Pull request closed without merging**